lib/PersonalityProvider.jsm

```diff
@@ -4,8 +4,41 @@
 "use strict";
 
 const {UserDomainAffinityProvider} = ChromeUtils.import("resource://activity-stream/lib/UserDomainAffinityProvider.jsm", {});
+const {PersistentCache} = ChromeUtils.import("resource://activity-stream/lib/PersistentCache.jsm", {});
+const {RemoteSettings} = ChromeUtils.import("resource://services-settings/remote-settings.js", {});
 
-// This is just a stub for now.
-this.PersonalityProvider = UserDomainAffinityProvider;
+/**
+ *
+ */
+this.PersonalityProvider = class PersonalityProvider extends UserDomainAffinityProvider {
+  // This is just a stub for now, extending UserDomainAffinityProvider until we flesh it out.
+  constructor(...args) {
+    super(...args);
+    this.interestVectorStore = new PersistentCache("interest-vector", true);
+  }
+
+  /**
+   * Get the nb or nmf collection from Remote Settings.
+   */
+  getModel(modelType) {
```

Comment:
Can you explain how this works a bit? I see talk about getting a collection, but then the test looks like it was checking for just one thing coming back. Is this returning a map of tags to models or what? That's super cool if it is, I'm just a bit confused.